uaclient/tests/test_config.py

```diff
@@ -27,6 +27,15 @@
 M_PATH = "uaclient.entitlements."
 
 
+RESP_ALL_RESOURCES_AVAILABLE = [
+    {"name": name, "available": True} for name in ENTITLEMENT_CLASS_BY_NAME
+]
+RESP_ONLY_FIPS_RESOURCE_AVAILABLE = [
+    {"name": name, "available": bool(name == "fips")}
```

Comment:
Total nit, but `name == "fips"` is already a bool so the call here is a noop. No need to fix here.